Userland/Applications/PixelPaint/Filters/HueAndSaturation.cpp

```diff
@@ -33,7 +33,7 @@ ErrorOr<RefPtr<GUI::Widget>> HueAndSaturation::get_settings_widget()
         auto settings_widget = GUI::Widget::construct();
         settings_widget->set_layout<GUI::VerticalBoxLayout>();
 
-        auto add_slider = [&](auto name, int min, int max, auto member) -> ErrorOr<void> {
+        auto add_slider = [&](StringView name, int min, int max, auto member) -> ErrorOr<void> {
```

Comment:
what's this got to do with Variant? and why is the last function parameter being declared auto okay?

Reply:
It is about a Variant inside a ErrorOr usually. I have no idea either what the exact conditions for this bug to trigger are.

Reply:
The bug is https://github.com/llvm/llvm-project/issues/67260